### PR TITLE
Dialog improvements and other misc code improvements

### PR DESCRIPTION
**Pull request by pedrozzz0**
Just minor stuff though.

### CODE REVIEW

File: scripts/run-container.sh [hunk 1]
```diff
@@ -54,29 +67,32 @@ fi
...
 
 # Graphics nodes permissions
-chmod 777 -R /dev/dri/*
-chmod 777 -R /dev/graphics/*
-chmod 777 -R /dev/fb*
+chgrp graphics /dev/dri/*
+chmod 666 -R /dev/dri/*
+chgrp graphics /dev/graphics/*
+chmod 666 -R /dev/graphics/*
+chmod 666 -R /dev/fb*
```
Comment (erfanoabdi) [inline scripts/run-container.sh:96]: i think we can make this graphics 660 too?

**Review by erfanoabdi — COMMENTED** [review]
Thanks but we're moving to python for setup and running

"rm -rf" recursive option added on some single files, any reason?

File: scripts/run-container.sh (hunk 1 above)
Comment (pedrozzz0) [reply scripts/run-container.sh:96]: tho, i'm not sure, cause dri was 666

**Comment by pedrozzz0** [comment]
> "rm -rf" recursive option added on some single files, any reason?

nope, just to make sure stuff will be deleted tho

File: scripts/run-container.sh [hunk 2]
```diff
@@ -21,7 +34,7 @@ start cgroup-lite
 umount -l /sys/fs/cgroup/schedtune
 
 # start sensors hal
-anbox-sensord &
+anbox-sensord
```
[the remark this answers is not in the document]
Comment (pedrozzz0) [reply scripts/run-container.sh]: fixed

**Comment by erfanoabdi** [comment]
thanks for your PR.
it's on deprecated branch, i have to close it

**Pull request closed without merging**